src/rpc/client.h

```diff
@@ -14,6 +14,12 @@ UniValue RPCConvertValues(const std::string& strMethod, const std::vector<std::s
 /** Convert named arguments to command-specific RPC representation */
 UniValue RPCConvertNamedValues(const std::string& strMethod, const std::vector<std::string>& strParams);
 
+/** Whether this is a positional blockhash parameter */
+bool RPCConvertBlockhash(const std::string& method, int pos);
+
+/** Whether this is a named blockhash parameter */
+bool RPCConvertNamedBlockhash(const std::string& method, std::string& param);
```

Comment:
Unused?

Reply:
Ooops. Added support for `-named` arguments in cli, so now it's used.